src/components/Tag/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { faTimes } from '@fortawesome/free-solid-svg-icons/faTimes';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Box, type BoxProps, Button, Tooltip, Typography } from '@mui/material';
+
+// Prevent the filters taking up too much horizontal space
```

Comment:
This comment does not make much sense in the context of the library alone, is there something a bit more generic we can change it to?